katas/test/src/pokerhands/joaquin/HandTest.cs

```diff
@@ -0,0 +1,55 @@
+namespace Hand.Test.Joaquin;
+
+using PokerHand.Joaquin;
+
+public class HandTest
+{
+    List<string> Cards = new List<string>()
+    {
+        {"AS 7H 8D JS QH"},
+        {"3C 3S JS 3H JH"},
+        {"7D 4D 6D 3D 5D"},
+        {"JS TD QH 2S 4C"},
+        {"5S AH 2C QS 4C"},
+    };
+
+    [Fact]
+    public void ItShouldCreateAHandOfPoker()
+    {
+        foreach (string card in Cards)
+        {
+            Hand hand = new Hand(card);
+            Assert.Equal(5, hand.Cards.Count);
+        }
+    }
+
+    [Fact]
+    public void ItShouldHaveValidValues()
+    {
+        Deck deck = new Deck();
+
+        foreach (string card in Cards)
+        {
+            Hand hand = new Hand("AS 7H 8D JS QH");
+            foreach (var c in hand.Cards)
+            {
+                Assert.Contains(c.CreateCard(), deck.cards);
+            }
+        }
+    }
+
+    // [Fact]
```

Comment:
Avoid leaving commented code if there is no good reason to keep it, helps to keep code clean

Reply:
Done.

Reply:
if you are sure that you will implement it later, lets leave it, otherwise, it could be better to remove it